session.py

```diff
@@ -0,0 +1,671 @@
+    def post_exercise(self, workoutId, exerciseday, exercises):
+        """ Makes a post request for an exercise
+
+            args:
+                workoutId( int ): workout id where you want to add the exercise
+                exerciseday( int ): training id where you want to add the exercise
+                exercises( int ): the specific id of the exercise
+
+            returns:
+                list: [response of the request, check(response)]
+        """
+        # create the URL
+        post_exerciseURL = self.base_URL + "set/"
+
+        # create referer and headers
+        exercise_post_headers = self.headers
+        post_exerciseReferer = self.base_workoutReferer + str(workoutId) + "/view/"
+        exercise_post_headers['Referer'] = post_exerciseReferer
+
+        # create the payload
+        payload_exercise = {
+            "exerciseday":exerciseday,
+            "exercises":exercises
+          }
+
+        # make the request
+        response_post_exercise = self.s.post(post_exerciseURL, json= payload_exercise ,headers= exercise_post_headers)
+        return [response_post_exercise, mySession.check(response_post_exercise)]
+
+    def post_nutritionplan(self):
+        """ Makes a post request for a nutritionplan
+
+            returns:
+                list: [response of the request, check(response)]
+        """
+
+        post_nutritionplanURL = self.base_URL + "nutritionplan/"
+        nutritionplan_post_headers = self.headers
+        post_nutritionplanReferer = self.base_nutritionReferer + "overview/"
+        nutritionplan_post_headers['Referer'] = post_nutritionplanReferer
+
+        response_post_nutritionplan = self.s.post(post_nutritionplanURL,headers= nutritionplan_post_headers)
+        return [response_post_nutritionplan, mySession.check(response_post_nutritionplan)]
+
+    def post_meal(self, nutritionplanId):
+        """ Makes a post request for a meal
+
+            args:
+                nutritionplanId( int ): nutritionplan id where you want to add the meal
+
+            returns:
+                list: [response of the request, check(response)]
+        """
+        # create the URL
+        post_mealURL = self.base_URL + "meal/"
+
+        # create referer and headers
+        meal_post_headers = self.headers
+        post_mealReferer = self.base_nutritionReferer + str(nutritionplanId) + "/view/"
+        meal_post_headers['Referer'] = post_mealReferer
+
+        # create the payload
+        payload_meal = {
+        "plan":nutritionplanId
+        }
+
+        # make the request
+        response_post_meal = self.s.post(post_mealURL, json = payload_meal , headers= meal_post_headers)
+        return [response_post_meal,mySession.check(response_post_meal)]
+    
+    def post_mealitem(self, nutritionplanId, mealId, ingredientId, amount):
+        """ Makes a post request for a meal item
+
+            args:
+                nutritionplanId( int ): nutritionplan id where you want to add the meal item
```

Comment:
Similar to the above comment. Nutrition plan ID of the meal in which you want to add the item.